posydon/popsyn/synthetic_population.py

```diff
@@ -230,6 +231,10 @@ def parse(self, S1_state=None, S2_state=None, binary_state=None,
                     df_sel_met = pd.concat([df_sel_met, df_tmp])
                     del df_tmp
 
+            if k != 0:
+                shift_index = max(np.unique(df_sel.index)) + 1 
+                df_sel_met.index += shift_index 
```

Comment:
is it important to keep the old index as a separate column?

Reply:
Also, is there an easy way to reindex them from 0-n_binaries in self.df where there are duplicate index? Right now the binary index can be very large which is unnecessary but convenient. I couldn't find something straightforward shifting the index